### ISSUE TITLE
Invalid identifier: Invalid characters supplied.

### ISSUE THREAD

**Issue by cjhaas** [paragonie/chronicle#58]
This was reported last year in https://github.com/paragonie/chronicle/issues/40 but that was related to scheduled tasks. Today I did a fresh install on Ubuntu 18.04 with PHP 7.3 and MariaDB 10.4.7 and am getting it with client creation.

When running:
```php bin/create-client.php -p "XXXXXX" -c "Test"```

The below is thrown:
```
PHP Fatal error:  Uncaught ParagonIE\EasyDB\Exception\InvalidIdentifier: Invalid identifier: Invalid characters supplied. in /home/chrishaas/chronicle/vendor/paragonie/easydb/src/EasyDB.php:297
Stack trace:
#0 /home/chrishaas/chronicle/vendor/paragonie/easydb/src/EasyDB.php(870): ParagonIE\EasyDB\EasyDB->escapeIdentifier('`chronicle_clie...')
#1 /home/chrishaas/chronicle/vendor/paragonie/easydb/src/EasyDB.php(519): ParagonIE\EasyDB\EasyDB->buildInsertQueryBoolSafe('`chronicle_clie...', Array)
#2 /home/chrishaas/chronicle/bin/create-client.php(126): ParagonIE\EasyDB\EasyDB->insert('`chronicle_clie...', Array)
#3 {main}
  thrown in /home/chrishaas/chronicle/vendor/paragonie/easydb/src/EasyDB.php on line 297
```

MariaDB version:
```
mysql  Ver 15.1 Distrib 10.4.7-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

PHP Version:
```
PHP 7.3.8-1+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Aug  7 2019 09:52:12) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.8, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.8-1+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies```

**Comment by paragonie-scott** [comment]
#59 should fix this.

**Comment by paragonie-scott** [comment]
This was fixed in `v1.1.5`
